### DISCUSSION TITLE
Done! Am I the first to actually make this thing?

### DISCUSSION THREAD

**Discussion by caschy** [FundamentalFrequency/.github#40] [Show and tell]
![IMG_20220702_194909036](https://user-images.githubusercontent.com/18508692/177019311-82f81542-359f-4516-a0ff-a3055a521bcd.jpg)
![IMG_20220702_194904274](https://user-images.githubusercontent.com/18508692/177019312-e542fa2d-2e79-461c-9b19-b5651bd8c106.jpg)
![IMG_20220702_194858326](https://user-images.githubusercontent.com/18508692/177019313-23940623-adcc-408a-ba91-1e9891ee5795.jpg)

**Comment by stonepreston** [comment]
Wow! Great job I love the color labels. Did you use a laser printer for those? Did you encounter any troubles along the way? Is the build guide pretty accurate?

**Reply by caschy** [reply]
Thanks! Yeah color laser. I think I have to spray them with some clear coat paint or nail polish or something cause they will probably eventually rub off.  Not sure if its my printer or what but the sticker tape stuff probably wont last.

Also, if you use a piece of acrylic of keys/switch layer, be careful when installing or it will crack.  Also don't crank down the m2.5 bolts or again, you'll crack the acrylic.  

Build guide was in general, pretty spot on for me.  

Also, for the hyperpixel screen, in order to get the touchscreen to be correctly oriented I had to change from touchscreen-inverted-y to touchscreen-inverted-x in the dtparam line for the boot/config.txt file.

In general, software needs some help.  I'm getting a lot of crashes and stuff.  I know its in alpha so I can't complain.

**Reply by stonepreston** [reply]
interesting, I havent noticed crashing like that. next time it happens get the logs from the .config/LMN-3 directory and open an issue in the DAW repository. Also yeah I must have copied the wrong screen config in the build guide regarding the inverted-y thing. Someone submitted a PR yesterday that fixed that.
